web-ng/src/app/components/card-view-project/card-view-project.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { CardViewProjectComponent } from './card-view-project.component';
+
+describe('CardViewProjectComponent', () => {
+  let component: CardViewProjectComponent;
+  let fixture: ComponentFixture<CardViewProjectComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ CardViewProjectComponent ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(CardViewProjectComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
```

Comment:
Please implement unit tests for new classes here and throughout.

Reply:
done.